src/main/kotlin/se/uulm/snowballr/backend/model/SnowballRException.kt

```diff
@@ -58,41 +58,48 @@ sealed class SnowballRException(
      *
      * @param currentUserId The ID of the user that is accessing the entity/entities.
      * @param accessedEntityMessage The message of what is accessed.
+     * @param accessType The type of access, i.e., an update, read access, ...
      */
     sealed class UnauthorizedException(
         currentUserId: String,
         accessedEntityMessage: String,
-    ) : SnowballRException("User with ID '$currentUserId' is not authorized to access $accessedEntityMessage") {
+        accessType: AccessType,
+    ) : SnowballRException("User with ID '$currentUserId' is not authorized to $accessType $accessedEntityMessage") {
         /**
          * Represents an [UnauthorizedException] that occurs when the current user accesses a single entity without
          * permission.
          *
          * @param accessedEntityType The type of the entity that was accessed without permission.
          * @param accessedEntityId The ID of the entity that was accessed without permission.
+         * @param accessType The type of access (@see [AccessType] for possible types).
```

Comment:
The `@see` annotation doesn't work that way, but you can just say 'see'

```suggestion
         * @param accessType The type of access (see [AccessType] for possible types).
```